godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/controllers.yaml

```diff
@@ -1,6 +1,5 @@
 controller_list:
-  - name: ""
-    action_ns: joint_trajectory_action
+  - name: 'joint_trajectory_action'
+    action_ns: ''
```

Comment:
It's good idea to create ros package, much better than before putting on
the godel folder.
I already added in my server, but I do have a simple question about the
github, How can I upload the whole folder in github?


About loading different rviz file based on namespace, below it is the steps
based on your suggestion, correct me if my understanding is correct or not.
1.  Create rviz file with different name pattern with $namespace
2. Load the related rviz file based on the pattern.

For this solution, user needs to manually copy and modify the rviz file,
and once the namespace changes, user needs to do this again, if there's any
arg inside the rviz to make this easier?

Thanks
Chencheng.




On Tue, Mar 8, 2016 at 11:17 AM, Jorge Nicho <notifications@github.com>
wrote:

> In
> godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/controllers.yaml
> <https://github.com/swri-robotics/godel/pull/1#discussion_r55392383>:
>
> > @@ -1,6 +1,5 @@
> >  controller_list:
> > -  - name: ""
> > -    action_ns: joint_trajectory_action
> > +  - name: 'joint_trajectory_action'
> > +    action_ns: ''
>
> On second thought, your script does set some environment variables
> required by ROS. Doing this from within a ros launch file might be
> difficult so I'm OK with keeping the shell script as it makes it easier to
> configure the environment for running ROS across multiple machines.
> What you can do is to put this script into its own ros package (e.g.
> godel_network_support ) so that you can use the rosrun command to call the
> script instead of having to cd into the source directory.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swri-robotics/godel/pull/1/files#r55392383>.
>